src/python/dxpy/bindings/dxapplet.py

```diff
@@ -222,8 +227,8 @@ def run(self, executable_input, project=None, folder=None, name=None, tags=None,
                                         debug=debug,
                                         delay_workspace_destruction=delay_workspace_destruction,
                                         priority=priority,
-                                        extra_args=extra_args)
-
+                                        extra_args=extra_args,
+                                        detach=detach)
```

Comment:
Here `detach` also should move before extra_args. And also please move it in the docstrings.